core/src/main/java/org/opensearch/sql/expression/ReferenceExpression.java

```diff
@@ -100,6 +103,17 @@ public ExprValue resolve(ExprTupleValue value) {
   }
 
   private ExprValue resolve(ExprValue value, List<String> paths) {
+    // This case is to allow returning all values in an array to be in one row
+    // and fixes https://github.com/opensearch-project/sql/issues/1305
```

Comment:
The resolved fixes may be better suited in the `Description` portion of the PR rather than the code.

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/213/commits/ff901da23fe331bfdad88f5d11cbb807c32c1b0c